library/elliptics.h

```diff
@@ -1035,8 +1035,7 @@ static inline int dnet_read_ll(int fd, char *data, size_t size, off_t offset)
 /*
  * Watermarks for number of bytes written into the wire
  */
-#define DNET_SERVER_SEND_WATERMARK_HIGH		(30*1024*1024*1024L)
-#define DNET_SERVER_SEND_WATERMARK_LOW		DNET_SERVER_SEND_WATERMARK_HIGH / 2
+#define DNET_SERVER_SEND_WATERMARK_HIGH		(100*1024*1024L)
```

Comment:
Does it mean that we cannot fully utilize 10Gb networks with fast raid/ssd destination disks?